src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.ts

```diff
@@ -107,11 +107,7 @@ export class RgwUserFormComponent extends CdForm implements OnInit {
       max_buckets_mode: [1],
       max_buckets: [
         1000,
-        [
-          CdValidators.requiredIf({ max_buckets_mode: '1' }),
-          CdValidators.number(false),
-          Validators.min(1)
```

Comment:
Why was 1 here and 0 in the template?

Reply:
Not exactly. Its 1 in the template too 
https://github.com/ceph/ceph/blob/d53261a5b050b880002094099dbd5e85b3218631/src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.html#L134-L138

The change I am making here is for `user_quota_max_objects` and `bucket_quota_max_objects`. But thanks for pointing it out, I'll need to remove the min validator here also.

Reply:
Done!